### PR TITLE
Fix many typos

### PR DESCRIPTION
**Pull request by goggle**
- This fix issue: 

## PR Description:

Corrects various typos that I've found in the repository. Please double-check that I haven't forgotten to adapt any method calls for the corrected method name definitions.

## Tests and Checks
- [ ] I have tested the code!<br>

### CODE REVIEW

**Comment by Torxed** [comment]
Please rebase this against `master`, as there are too many changes for me to tackle.
Apologies for this, had to prioritize testing and releasing of the new version. Renaming things could cause delays in testing.

**Comment by svartkanin** [comment]
@goggle any chance you could update the PR with latest master?

**Comment by goggle** [comment]
@Torxed @svartkanin Here is an updated version of this PR.

File: archinstall/lib/global_menu.py [hunk 1]
```diff
@@ -119,7 +119,7 @@ def setup_selection_menu_options(self):
 		self._menu_options['parallel downloads'] = \
 			Selector(
 				_('Parallel Downloads'),
-				lambda preset: add_number_of_parrallel_downloads(preset),
+				lambda preset: add_number_of_parallel_downloads(preset),
```
Comment (erickisos) [inline archinstall/lib/global_menu.py:122]: Can we pass the method reference instead of a lambda?
```suggestion
				add_number_of_parallel_downloads,
```

**Review by erickisos — COMMENTED** [review]
I was randomly reading some PRs and found yours, looks good tbh

File: archinstall/lib/global_menu.py (hunk 1 above)
Comment (Torxed) [reply archinstall/lib/global_menu.py:122]: Will the `preset` be preserved in the way we call the function at a later stage?

**Review by Torxed — APPROVED** [review]

File: archinstall/lib/global_menu.py (hunk 1 above)
Comment (svartkanin) [reply archinstall/lib/global_menu.py:122]: we do it the same way with most other settings as well, I don't think it makes much difference

**Pull request merged**